### PR TITLE
added github issue doc

### PR DESCRIPTION
**Pull request by corpetty**

### CODE REVIEW

File: process/github-issues.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Security Process Using Github Issues (DRAFT)
+This document outlines the process of using Github issues to coordinate security issues of a given project.
+
+## Should you use Github issues?
+ Before submitting any issues, the disclosure should have already engaged in a conversation with the Status Security team about appropriate methods of disclosure for a given project. This should be described in the SECURITY.md file within any given repository. If it is not, then follow the advice of this repository's README.md. If the discloser is still unsure, then please request guidance at security@status.im.
```
Comment (maurelian) [inline process/github-issues.md]: type '..., the discloser...'

File: process/github-issues.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+Once the discloser is aware that Github issues are the appropriate methodology of disclosure, the following procedure is **RECOMMENDED****:
+- A github issue **MUST** be created for ever individual assessment discovery
+- a `difficulty` label **MUST** be added to every issue
+    - The following options are available:
+        - `difficulty:low`
+        - `difficulty:medium`
+        - `difficulty:high`
+        - `difficulty:undetermined`
+- a `severity` label **MUST** be added to every issue
+    - The following options are available:
```
Comment (maurelian) [inline process/github-issues.md]: When should one choose `informational` and `does-not-spark-joy` (great name btw).
Any guidance on assigning severity?

File: process/github-issues.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# Security Process Using Github Issues (DRAFT)
+This document outlines the process of using Github issues to coordinate security issues of a given project.
+
+## Should you use Github issues?
+ Before submitting any issues, the disclosure should have already engaged in a conversation with the Status Security team about appropriate methods of disclosure for a given project. This should be described in the SECURITY.md file within any given repository. If it is not, then follow the advice of this repository's README.md. If the discloser is still unsure, then please request guidance at security@status.im. 
+
+## Process
+Once the discloser is aware that Github issues are the appropriate methodology of disclosure, the following procedure is **RECOMMENDED****:
+- A github issue **MUST** be created for ever individual assessment discovery
+- a `difficulty` label **MUST** be added to every issue
```
Comment (maurelian) [inline process/github-issues.md]: Difficulty of... identifying the issue?

File: process/github-issues.md (hunk 2 above)
Comment (corpetty) [reply process/github-issues.md]: lol, that comes from @lojikil and their audits.  I think it's kind of a label for "nothing is wrong but I don't like it, it should probably be done differently."  I'll let him comment on it exactly

File: process/github-issues.md (hunk 3 above)
Comment (corpetty) [reply process/github-issues.md]: this should be specified, difficulty of successfully performing the issue at hand.

Comment (maurelian) [reply process/github-issues.md]: ie. difficulty of exploiting the vulnerability?

File: process/github-issues.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
+# Security Process Using Github Issues (DRAFT)
+This document outlines the process of using Github issues to coordinate security issues of a given project.
+
```
Comment (mratsim) [inline process/github-issues.md]: Add

```Markdown
## Requirements

   The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
   "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this
   document are to be interpreted as described in [RFC2119](https://www.ietf.org/rfc/rfc2119.txt)

File: process/github-issues.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+        - `status:wont-fix`
+        - `status:resolved`
+- a `type` label **MUST** be added to every issue
+    - The following options are available:
+        - `type:docs`
+        - `type:bug`
+        - `type:arch`
+        - `type:enhancement`
+- a Status owner **SHOULD** be named as decided by the security coordinator to every issue
+- a Vender owner **SHOULD** be named as decided by the vendor to every issue
```
Comment (mratsim) [inline process/github-issues.md]: Vendor

File: process/github-issues.md (hunk 4 above)
Comment (mratsim) [reply process/github-issues.md]: Scratch that it's at the end

File: process/github-issues.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    - The following options are available:
+        - `difficulty:low`
+        - `difficulty:medium`
+        - `difficulty:high`
+        - `difficulty:undetermined`
+- a `severity` label **MUST** be added to every issue
+    - The following options are available:
+        - `severity:informational`
+        - `severity:does-not-spark-joy`
+        - `severity:low`
```
Comment (kdeme) [inline process/github-issues.md]: Would it be useful to add a link like e.g. [this](https://www.first.org/cvss/calculator/3.0) to guide in the selection of severity level?

Comment (corpetty) [reply process/github-issues.md]: yes

File: process/github-issues.md (hunk 3 above)
Comment (corpetty) [reply process/github-issues.md]: yes

**Pull request merged**